x-pack/plugins/lists/common/shared_exports.ts

```diff
@@ -36,6 +36,7 @@ export {
   listSchema,
   entry,
   entriesNested,
+  nestedEntries,
```

Comment:
...but then you see it here with another already existing schema name and it seems a bit wonky. Any thoughts?

Reply:
changed to `nestedEntryItem` thanks @dhurley14